catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/impl/FileSystemPersistenceProvider.java

```diff
@@ -154,9 +154,10 @@ Object loadFromPersistence(String key) {
         try {
             inputStream = new FileInputStream(getMapStorePath() + key + SER);
             InputStream buffer = new BufferedInputStream(inputStream);
-            ObjectInput input = new ObjectInputStream(buffer);
-            Object obj = (Object) input.readObject();
-            return obj;
+            try(ObjectInput input = new ObjectInputStream(buffer)) {
+                Object obj = (Object) input.readObject();
```

Comment:
The least necessary cast in the history of casts. :)

And this could simply be an inline return as well.